app/views/support_interface/courses/show.html.erb

```diff
@@ -18,6 +18,14 @@
   'Start on Apply' => govuk_link_to('View on Apply (candidate)', candidate_interface_apply_from_find_path(providerCode: @course.provider.code, courseCode: @course.code)),
 })) %>
 
+<h2 class='govuk-heading-m'>Vacancies</h2>
+
+<%= render(SupportInterface::CourseChoicesTableComponent.new(course_options: @course.course_options)) %>
+
+<h2 class='govuk-heading-m'>Applications</h2>
+
+<%= render SupportInterface::ApplicationsTableComponent.new(application_forms: @course.application_choices.map(&:application_form).uniq) %>
```

Comment:
There's a potential 1+n query here I think. `ApplicationForm.joins(application_choices: :course_option).where(application_choices: { course_options: { course: @course } }).distinct` would avoid this but there may be neater ways of doing it.

Reply:
Fixed!